contribs/emissions/src/main/java/org/matsim/contrib/emissions/analysis/EmissionsOnLinkEventHandler.java

```diff
@@ -86,5 +93,13 @@ private void handleEmissionEvent(double time, Id<Link> linkId, Map<Pollutant, Do
         } else{
             currentBin.getValue().get( linkId ).addEmissions( emissions );
         }
+
```

Comment:
This could be simplified like: 

```
var linkEmissions = link2pollutants.computeIfAbsent(linkId, id -> new HashMap<>());
for (var entry : map.entrySet()) {
    linkEmissions.merge(entry.getKey(), entry.getValue(), Double::sum);
}